Makefile

```
@@ -78,7 +78,7 @@ generate: keyservice/keyservice.pb.go
 	$(GO) generate
 
 %.pb.go: %.proto
-	protoc --go_out=plugins=grpc:. $<
+	protoc --go_grpc_opt=require_unimplemented_servers=false --go_grpc_out=. --go_out=. $<
```

I can't get this working with `go_grpc`, I have to use `go-grpc`:
```suggestion
	protoc --go-grpc_opt=require_unimplemented_servers=false --go-grpc_out=. --go_out=. $<
```
I installed it with `go install google.golang.org/grpc/cmd/protoc-gen-go-grpc@latest` (and `go install google.golang.org/protobuf/cmd/protoc-gen-go@latest`).

(With the latest version the generated files also look differently.)

---

I'll take another stab at this then.